src/holo-files/main_test.go

```diff
@@ -0,0 +1 @@
+../main.go.test
```

Comment:
The binaries produced by `go test` don't actually call `main()`; instead, you need to define a test that calls your main function.